src/views/Chat/Chat.jsx

```diff
@@ -24,12 +24,18 @@ export default function Chat() {
   };
 
   useEffect(() => {
-    getMessages().then(setMessages);
-    // TODO: Subscribe to message changes, using handleMessageReceived
-    // as the callback function
-    // Don't forget to return a cleanup method (`unsubscribe`)
-    // from this useEffect!
-  }, []);
+    getMessages()
+      .then(setMessages)
+      .then(subscribe)
+
```

Comment:
If we checkout our subscribe method, it actually takes a callback function as a parameter. Be sure to call our handleMessageReceived function as a parameter.